cogs/slowmode.py

```diff
@@ -21,14 +41,57 @@ def __init__(self, client):
     @commands.has_any_role(MOD_ROLE, HELPER_ROLE)
     async def slowmode(self, ctx: commands.Context, interval_str: str):
         '''Activate slowmode in help channels'''
-        # TODO: check against channel ID instead of name
         interval = util.get_id_from_tag(interval_str)
-        if isinstance(ctx.channel, discord.TextChannel) and \
-                ctx.channel.name in ['kb-help', 'kb-buying-advice']:
+        if isinstance(ctx.channel, discord.TextChannel) and ctx.channel.id in CHANNELS:
             await ctx.channel.edit(slowmode_delay=interval)
             await ctx.channel.send("Slow it down!")
 
 
+    def get_delay(self, message_count):
+        message_limits = sorted(time_configs, reverse=True)
+        for limit in message_limits:
+            if message_count >= limit:
+                return time_configs[limit]
+        return 0
+
+
+    async def update_slowmode(self):
```

Comment:
need a way to disable this if helpers think this needs configuration changes

Reply:
Should I create a new command to disable it, or should I integrate it into the current slowmode command?

Reply:
added `self.disabled`, will make it configurable with a command